tests/ctverif/count_success.pl

```diff
@@ -20,18 +20,32 @@
 use strict;
 use warnings;
 
+sub  trim { my $s = shift; $s =~ s/^\s+|\s+$//g; return $s };
+
+
 if (@ARGV != 2) {
     die "usage: count_success.pl expected_success expected_failures";
 }
 
 my $expected_success = shift;
 my $expected_failure = shift;
+my @undefined_functions = ();
 
 my $verified = 0;
 my $errors = 0;
 while (my $line = <STDIN>){
     print $line;
-    if ($line =~ /Boogie program verifier finished with (\d+) verified, (\d+) errors/) {
+    #Check if the code under test used unexpected functions
+    if ($line =~ /warning: module contains undefined functions:([a-zA-Z0-9_, ]+)/) {
+	for my $fns (split(",",$1)){
+	    my $trimmed = trim ($fns);
+	    push @undefined_functions, $trimmed;
+	    print "found some !!!!!\n";
```

Comment:
If we're already printing out all of the functions at the end, I don't think we need this line.